xml/vt_scenarios.xml

```diff
@@ -0,0 +1,198 @@
+ <title>Virtualization Scenarios</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker>
+   </dm:bugtracker>
+  </dm:docmanager>
+  <abstract>
+   <para>
+    Virtualization design provides several useful capabilities to your
+    organization: more efficient hardware use, support for legacy software,
+    operating system isolation, live migrations, disaster recovery, and 
+    load balancing. 
+   </para>
+  </abstract>
+ </info>
+ <sect1 xml:id="sec-virtualization-scenarios-capabilities">
+  <title>Server Consolidation</title>
+  <para>
+   Many servers can be replaced by one big physical
+   server, so hardware is consolidated, and guest operating systems are
+   converted to virtual machines. This also supports running legacy
+   software on new hardware.
+   <!-- reduce cost -->
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Better usage of resources which were not running at 100%
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Fewer servers locations needed
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Compute resources more efficiently: multiple workloads on the same server
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Simplification of the data center infrastructure
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Moving workload is easier to another host, this will avoid service downtime 
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Virtual Machine provisioning
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Multiple guest operating system will be able to run on 1 host
+    </para>
+   </listitem>
+  </itemizedlist>
+  <important>
+   <para>The server consolidation scenario needs special attention.</para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      Maintenance windows should be carefully planned
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Storage is a key point to be able to support migration and growing disk usage
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Server capabilities must be checked to support the new workload
+     </para>
+    </listitem>
+   </itemizedlist>
+  </important>
+ </sect1>
+ <sect1>
+  <title>Isolation</title>
+  <para>
+   A guest operating system can be fully isolated from the host
+   running it. So if the virtual machine is corrupted, the host system is
+   not harmed. It also does not affect any other virtual machines. No data
+   is shared between VM.
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Secure Boot can be used for VM
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     KSM should be avoided
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     CPU can be assigned to a VM, and HT should be disabled to avoid security issue
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     VM should not share network / storage / network / Hardware
```

Comment:
This seems much too sweeping. It is both unrealistic and could be undesirable. The entire point of consolidation via virtualisation is that VMs (note plural) share all these things.

I suggest removing this line.

Reply:
Its the Isolation scenarios. We should have a note which explains that VM could not share anything between them.

Reply:
There is an important difference between sharing resources between VMs, and sharing host resources between VMs.

A modern high-end server could run 20-30 well-specced VMs or more simultaneously, and many more is possible.

It is not realistic to equip the server with 20 or 30 separate Ethernet cards -- so VMs must share those.

They will often share CPUs. They will always share the host's memory space. Their disk images will be stored on the  host's disk drive or array. So these resources will normally be shared, and this is explicitly stated as a good thing under "Consolidation".

Reply:
You can confine a VM guest's CPU and memory to a NUMA node. You can confine a network card usage to one guest only. You can restrict access to storage to a specific VM. Consolidation and Isolation are different scenario, you can not really mix them, its a matter of compromise. Strong isolation is possible if required, we are not speaking of efficienty, good usage of resources or whatever.

Reply:
Then you need to spell this out, because at the moment, one paragraph says "consolidating via sharing resources" and a few paragraphs later it says "do not share resources ever because it is insecure!"

I think the idea of dedicated hardware network interfaces is highly unrealistic, TBH. That's what things like VLANs are for. The main reason for dedicated interfaces on networks these days, as far as I know, is performance and robustness. E.g. dedicated channels for node-to-node comms, or cluster-admin to nodes, away from the network carrying workloads. Or dedicated networks for SAN traffic, away from public-facing client networks.

This is a complex area, and trying to simplify it down to a couple of bullet points will add confusion. Either we need to do the legwork to make the different scenarios clear, or we need to remove this duplication of topics with conflicting meanings. There can be no middle ground without removing clarity and lessening the info we are conveying.

Reply:
We can improve this later. PM and TL proposed anothr way to present this list, with workloads approach. This will be done in a next version.